quality/README.md

```diff
@@ -67,3 +67,22 @@ jacocoAndroidUnitTestReport {
     xml.enabled false
 }
 ```
+
+## Errorprone (with NullAway)
+// root script
+buildscript {
+  apply from: "config/index.gradle"
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
+  }
```

Comment:
have you tried putting it into the `quality/errorprone/android.gradle` using the `plugin` dsl?
sometimes that worked, sometimes it doesn't. would be cleaner if that works.

Reply:
I tried plugin first, it's not working.

Reply:
:(